web/themes/custom/nica_theme/less/modules/variables-modules.less

```diff
@@ -0,0 +1 @@
+@background-image: none;
```

Comment:
ok!!!


2016-08-26 14:44 GMT-06:00 Lucas Hedding <notifications@github.com>:

> In web/themes/custom/nica_theme/less/modules/variables-modules.less
> <https://github.com/MTechLLC/nica/pull/94#discussion_r76485273>:
>
> > @@ -0,0 +1 @@
> > +@background-image: none;
>
> Igual. No background no es global.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MTechLLC/nica/pull/94/files/ef1cc2b1f089c24037284a21439bff06215ba099#r76485273>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATtl_XNejSs5OCwcLSNdT3TpsTNdd1Bqks5qj1BBgaJpZM4JuaeY>
> .
>